src/Connection.hpp

```diff
@@ -38,6 +38,11 @@ class Connection
 	size_t _responseSize;
 	size_t _responseSizeSent;
 	std::string _responseString;
+	bool _hasCGI;
+	bool _CGIHasExited;
+	pid_t _CGIPid;
+	int _CGIExitStatus;
+	time_t _CGIStartTime;
```

Comment:
Maybe it would be better to make a CGI struct, but it is a small design thing.

Reply:
I took a note: https://linear.app/web-serv/issue/WEB-274/create-a-cgi-struct-inside-connection